tests/b006_b008.py

```diff
@@ -42,35 +73,61 @@ def this_too(value=collections.OrderedDict()):
     ...
 
 
-async def async_this_too(value=collections.OrderedDict()):
+async def async_this_too(value=collections.defaultdict()):
+    ...
+
+
+def dont_forget_me(value=collections.deque()):
     ...
 
 
+# N.B. we're also flagging the function call in the comprehension
+def list_comprehension_also_not_okay(default=[i**2 for i in range(3)]):
+    pass
+
+
+def dict_comprehension_also_not_okay(default={i: i**2 for i in range(3)}):
+    pass
+
+
+def set_comprehension_also_not_okay(default={i**2 for i in range(3)}):
+    pass
+
+
+def kwonlyargs_mutable(*, value=[]):
+    ...
+
+
+# Recommended approach for mutable defaults
 def do_this_instead(value=None):
     if value is None:
         value = set()
 
 
+# B008
+# Flag function calls as default args (including if they are part of a sub-expression)
 def in_fact_all_calls_are_wrong(value=time.time()):
     ...
 
 
-LOGGER = logging.getLogger(__name__)
+def f(when=dt.datetime.now() + dt.timedelta(days=7)):
+    pass
 
 
-def do_this_instead_of_calls_in_defaults(logger=LOGGER):
-    # That makes it more obvious that this one value is reused.
+def can_even_catch_lambdas(a=(lambda x: x)()):
     ...
 
 
-def kwonlyargs_immutable(*, value=()):
-    ...
+# Recommended approach for function calls as default args
+LOGGER = logging.getLogger(__name__)
 
 
-def kwonlyargs_mutable(*, value=[]):
```

Comment:
Do we need an empty list for coverage? I guess you removed this cause of all the list comprehension tests now ...